tracing-fmt/src/default.rs

```diff
@@ -90,8 +100,8 @@ where
         write!(
             writer,
             "{} {}{}: ",
-            FmtLevel(meta.level()),
-            FullCtx(&ctx),
+            FmtLevel(meta.level(), self.ansi),
```

Comment:
Instead of passing a bool, we could have a marker type on `FmtLevel` and `FmtCtx`, and they could have different impls based on that?